README.md

```diff
@@ -1,59 +1,65 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Space Travelers' Hub
 
-> One paragraph statement about the project.
+> Single-page application that displays SpaceX rockets and missions.
 
 ![screenshot](./app_screenshot.png)
 
```

Comment:
[OPTIONAL] I will suggest adding the latest version (header active and hover styling) so looks familiar, don't you think?